src/pages/index.tsx

```diff
@@ -1,55 +1,51 @@
+import { ContentPageRenderer } from '@meemoo/admin-core-ui';
 import { GetServerSidePropsResult, NextPage } from 'next';
 import { useRouter } from 'next/router';
 import { GetServerSidePropsContext } from 'next/types';
-import { useEffect } from 'react';
+import { ComponentType } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
-import { BooleanParam, StringParam, useQueryParams, withDefault } from 'use-query-params';
+import { BooleanParam, useQueryParams } from 'use-query-params';
 
-import { Permission } from '@account/const';
+import { withAdminCoreConfig } from '@admin/wrappers/with-admin-core-config';
 import { AuthModal } from '@auth/components';
-import { selectHasCheckedLogin, selectIsLoggedIn, selectUser } from '@auth/store/user';
-import LoggedInHome from '@home/components/LoggedInHome/LoggedInHome';
-import LoggedOutHome from '@home/components/LoggedOutHome/LoggedOutHome';
-import { SHOW_AUTH_QUERY_KEY, VISITOR_SPACE_SLUG_QUERY_KEY } from '@home/const';
+import { selectUser } from '@auth/store/user';
+import { SHOW_AUTH_QUERY_KEY } from '@home/const';
 import { Loading } from '@shared/components';
 import { getDefaultServerSideProps } from '@shared/helpers/get-default-server-side-props';
-import { useHasAllPermission } from '@shared/hooks/has-permission';
+import { renderOgTags } from '@shared/helpers/render-og-tags';
 import { useNavigationBorder } from '@shared/hooks/use-navigation-border';
 import { selectShowAuthModal, setShowAuthModal } from '@shared/store/ui';
 import { DefaultSeoInfo } from '@shared/types/seo';
-import { isBrowser } from '@shared/utils';
 
-import VisitorLayout from '../modules/visitors/layouts/VisitorLayout/VisitorLayout';
+import { useGetContentPage } from '../modules/content-page/hooks/get-content-page';
+import { ContentPageService } from '../modules/content-page/services/content-page.service';
```

Comment:
You should use the admin-core-ui version of this service, since in one of Silkes pending PR's this content service is removed.
```
import { ContentPageRenderer, ContentPageService } from '@meemoo/admin-core-ui';
```